tests/provision/bootc/data/plans.fmf

```diff
@@ -15,15 +15,15 @@ execute:
             add-tmt-dependencies: true
             container-image: localhost/tmt-bootc-needs-deps
         environment:
-            PATTERN: localhost/tmtmodified
+            PATTERN: localhost/tmt/bootc/
```

Comment:
Since the variable value is now shared across all plans, it can be defined just once on the root level:
```
discover:
    how: fmf
provision:
    how: bootc
    disk: 20
execute:
    how: tmt
environment:
    PATTERN: localhost/tmt/bootc/
```